django_mailbox/models.py

```diff
@@ -498,10 +498,11 @@ def reply(self, message):
         pre-set it.
 
         """
-        if self.mailbox.from_email:
-            message.from_email = self.mailbox.from_email
-        else:
-            message.from_email = settings.DEFAULT_FROM_EMAIL
+        if not getattr(message, 'from_email', None):
```

Comment:
It might be slightly better if you instead used `if not hasattr(message, 'from_email')`.